documentation/asciidoc/computers/compute-module/cmio-display.adoc

```diff
@@ -61,3 +61,21 @@ To entirely ignore the display when connected, add the following line to `/boot/
 ----
 ignore_lcd=1
 ----
+
+== Attaching the Touch Display 2 LCD panel
+
+Touch Display 2 is a 720p 7" LCD display designed specifailly for Raspberry Pi devices (see https://www.raspberrypi.com/products/touch-display-2/). It connects in much the same way as the original touch display, but the software setup on Compute Modules is slightly different as it uses a different display driver. See xref:../../accessories/touch-display-2/about.adoc[Touch Display 2] for connection details.
+
+Edit the /boot./firmware/config.txt file and add the following to enable Touch Display 2 on DSI1.
+
+[source,ini]
+----
+dtoverlay=vc4-kms-dsi-ili9881-7inch
```

Comment:
@6by9 Are these instructions equally applicable to CM3 / CM4 / CM5 ?

Reply:
Yes.

CM1&3 requires the extra wiring for I2C to the display connector. CM4 and 5 need the jumpers on J6 where necessary. I think those are covered already.